### PR TITLE
Update LICENSE from LGPL 2.0 to Apache 2.0

### PR DESCRIPTION
**Pull request by danyeaw**
Signed-off-by: Dan Yeaw <dan@yeaw.me>

Migrates the project from LGPL 2.0 to Apache 2.0

[ ] One file not update: gaphor/ui/layout.py says that it is from etkdocking and is GPL.

### PR Checklist
Please check if your PR fulfills the following requirements:

- [X] I have read, and am following the [Contributer guide](https://github.com/gaphor/gaphor/blob/master/CONTRIBUTING.md)
- [X] I have read and understand the [Code of Conduct](https://github.com/gaphor/gaphor/blob/master/CODE_OF_CONDUCT.md)

### PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [X] License change

Issue Number: #61 

### Does this PR introduce a breaking change?
- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by danyeaw** [comment]
I have been thinking about this more. In the future I would like to move towards the project using Toga for the GUI toolkit instead of GTK. Until that time, it might be more strategic to keep the project LGPL. GPL code can't be used in an Apache project at all, and most of the GTK exampes or apps we can borrow ideas from are GPL.

**Comment by amolenaar** [comment]
I do not really consider that a problem. Anyhow, Apachev2 and GPLv2 are considered incompatible: https://apache.org/licenses/GPL-compatibility.html.

**Comment by danyeaw** [comment]
@amolenaar It is ok with me. I think we need to update gaphor/ui/layout.py and the win-installer to remove GPL code before we merge this though.

**Comment by amolenaar** [comment]
I can check with @dieterv.

**Comment by dieterv** [comment]
@amolenaar: i haven't had the luxury of time to work on etk for a long while now and unfortunately it doesn't look like that will change anytime soon. I'm willing to relicence everything to whatever licence you prefer and formally donate the code and ownership to the gaphor project if that helps you guys.

**Comment by amolenaar** [comment]
We removed most of the etk.docking code, since it was hard to port to GTK+3. We just copied a few bits that make the page loading somewhat easier from etk.docking into Gaphor.

I'll change the copyright notice.

Thanks for replying so quicky 😄

**Comment by dieterv** [comment]
Yeah, loads of changes in the dnd machinery for starters... Anyway, you're welcome 😊

File: README.md [hunk 1]
```diff
@@ -190,8 +190,8 @@ See [the contributing file](CONTRIBUTING.md)!
 ## :copyright: License
 Copyright (C) Arjan Molenaar and Dan Yeaw
 
-Licensed under the [Library General Public License 2.0](LICENSE.txt).
-
-Summary: You can copy, distribute, or modify Gaphor, as long as you include the
-required notices and license them under the LGPL.
+Licensed under the [Apache License v2](LICENSE.txt).
```
Comment (amolenaar) [inline README.md]: Should link to `LICENSE.md`

Comment (amolenaar) [reply README.md]: Fixed it :)

**Pull request merged**